src/main/io/beeper.c

```diff
@@ -338,11 +340,20 @@ void beeperUpdate(timeUs_t currentTimeUs)
         return;
     }
 
+    #ifdef USE_DSHOT
+    if (!ARMING_FLAG(ARMED) && currentBeeperEntry->mode == BEEPER_RX_SET) {  
```

Comment:
This sends the dshot beep command when not armed and with the beeper mode switch flipped